.github/workflows/sanity_checks.yml

```diff
@@ -40,6 +56,9 @@ jobs:
 
       - uses: ilammy/msvc-dev-cmd@v1
       - name: Sanity Checks
+        env:
+          CC: clang-cl
+          CXX: clang-cl
```

Comment:
Same here?

Reply:
IIRC using a matrix messes up the Jobs view as the jobs get grouped together. See here for an example: https://github.com/mesonbuild/wrapdb/actions/runs/4603869421

Reply:
Oh, I didn't think of that. I know the side bar view shows them all in a column, and that's the part I usually look at.

Reply:
If that view is acceptable I'll revert back.

Reply:
Honestly I don't hugely care. Which one do you prefer?

Reply:
current version. I usually click the middle of the screen to get to a specific run. one less click than with a matrix.

Reply:
Cool, then let's keep it.